### DISCUSSION TITLE
The material-ui library is imported but not used.

### DISCUSSION THREAD

**Discussion by mendelssohnbach** [react-hook-form/react-hook-form#5175] [Q&A]
I am learning react-hook-form.
Thanks for publishing meaningful information.

https://react-hook-form.com/jp/get-started#IntegratingwithUIlibraries

Input for lowercase letters may be a typo.

```
<input  // typo?

<Input  // mean?
```

```
import Input from "@material-ui/core/Input";

        render={({ field }) => <input {...field} />}
      />

      <input type="submit" />
```

**Comment by bluebill1049** [comment]
oh i see, will fix that.

**Reply by mendelssohnbach** [reply]
Thank you bluebill1049

I have confirmed that it has been fixed.

However, the formatter may not be behaving as expected.

https://react-hook-form.com/jp/get-started#Integratinganexistingform
Integrating with global state

The three buttons are not displayed.

```
CodeArea-module--button--  // JS Button

CodeArea-module--button--  // TS Button

CodeArea-module--button--  // CodeSandbox Button
```

---

This code sample also seems to have been converted to uppercase.

https://react-hook-form.com/get-started/#Handleerrors
Handle errors

```
import React from "react";
import { useForm } from "react-hook-form";

      <Input {...register("firstName", { required: true })} />
      
      <Input {...register("lastName", { required: true })} />
```

**Reply by bluebill1049** [reply]
thanks again! i don't have csb for the redux one, but I have adjusted the other issue.
